components/tests/ui/testcases/web/rdef_test.txt

```diff
@@ -148,9 +159,9 @@ Test Owners Rdef
     Go To                                   ${WELCOME URL}
     Select Experimenter
     Select And Expand Image
-    ${imageId}=                             Wait For Image Panel
+    ${imageId}=                             Wait For General Panel And Return Id      Image
     Click Link                              Preview
-    Wait For Preview Load
+    ${status}    ${oldId}    Wait For Preview Load      FAIL    '1'
```

Comment:
@will-moore that is how robot framework keyword return multiple variable. if it helps I can change ``${oldId}=`` to ``${imageId}=`` but that is all. We need ``${status}`` as well.